package.json

```diff
@@ -52,7 +52,8 @@
   },
   "devDependencies": {
     "@adobe/eslint-config-helix": "1.1.3",
-    "@adobe/helix-deploy": "3.0.8",
+    "@adobe/helix-deploy": "3.0.9",
+    "@pollyjs/adapter-fetch": "5.1.0",
```

Comment:
@tripodsan Why? This adapter is not used and apart from that would only intercept `fetch` in the `global` namespace.

Reply:
wasn't intentional. probably a merge from master.